data/json/regional_map_settings.json

```
@@ -729,6 +729,7 @@
           "mx_science": 400,
           "mx_collegekids": 500,
           "mx_roadblock": 1000,
+          "mx_roadblock_mil": 75,
```

To match the 1/6 ratio of the hardcoded mx_roadblock, this should be
```suggestion
          "mx_roadblock": 830,
          "mx_roadblock_mil": 170,
```